app_dart/lib/src/request_handlers/file_flaky_issue_and_pr.dart

```diff
@@ -99,7 +99,8 @@ class FileFlakyIssueAndPR extends ApiRequestHandler<Body> {
       return true;
     }
     // Skips if the flaky percentage is below the threshold.
-    if (statistic.flakyRate < _threshold) {
+    final threshold = ciYaml.getFirstTarget(statistic.name)?.flakinessThreshold ?? _threshold;
```

Comment:
It looks like `getIgnoreFlakiness` already has a pattern like this that uses `postsubmitTargets` instead of your new `getFirstTarget`?
https://github.com/flutter/cocoon/blob/4369cde1522bf88dfc2b6f99fb62c8a45108ae3c/app_dart/lib/src/request_handlers/check_flaky_builders.dart#L168-L172

Reply:
I think you're right about using post submits, will update. I do need this to work in multiple places; so making another getter here wouldn't look great.

I'll refactor.